packages/appservice-rest/src/Utilities/KuduServiceUtility.ts

```diff
@@ -201,11 +201,14 @@ export class KuduServiceUtility {
     public async deployWebAppImage(appName: string, images: string, isLinux: boolean) {
         try {
             core.debug(`DeployWebAppImage - appName: ${appName}; images: ${images}; isLinux:${isLinux}`);
-            if (!isLinux) {
-                throw new Error("Windows Containerized web app is not available for Publish profile auth scheme.");
-            }
             console.log(`Deploying image ${images} to App Service ${appName}`);
-            let headers = {'LinuxFxVersion': `DOCKER|${images}`};
+

```

Comment:
Please upgrade version in package.json https://github.com/microsoft/pipelines-appservice-lib/blob/1af035a49f04cdc63ed8fbd5790b235a32ec03a7/packages/appservice-rest/package.json

Reply:
Updated to 1.3.2.